invenio_rdm_records/services/schemas/parent/access.py

```diff
@@ -70,12 +70,17 @@ class AccessSettingsSchema(Schema):
     allow_user_requests = fields.Boolean()
     allow_guest_requests = fields.Boolean()
 
-    accept_conditions_text = SanitizedHTML()
+    accept_conditions_text = SanitizedHTML(allow_none=True)
 
 
 class ParentAccessSchema(Schema, FieldPermissionsMixin):
     """Access schema."""
 
+    class Meta:
+        """Meta attributes for the schema."""
+
+        unknown = EXCLUDE
```

Comment:
we should not need this? What was the case?

Reply:
The fields that are marked as `dump_only=True` are considered as the `unknown` fields during deserialisation and the default behaviour of the marshmallow schema validation is to raise the `Unknown field` error. 
The `unknown = EXCLUDE` prevents it from this and just excludes those fields. Same is happening on the record schema here https://github.com/inveniosoftware/invenio-rdm-records/blob/0ec37706559185d28899d0304ca090df2e93953c/invenio_rdm_records/services/schemas/record.py#L41
But it is not applied for the nested schemas by itlself unfortunately

Reply:
we have a utility for this https://github.com/inveniosoftware/invenio-rdm-records/blob/d1f55f95bde32963357eba8127acd06b82ea7be9/invenio_rdm_records/services/schemas/parent/__init__.py#L51

Reply:
Since another schema (`AccessSettingsSchema`) was used for deserializing the record, this is not needed anymore